signin.html

```diff
@@ -23,21 +23,27 @@
         </div>
       </header>
         <form class=" login">
+          <div class="form-group">
           <label>이메일</label>
           <input
             id="email"
             name="email"
             type="email"
             placeholder="이메일을 입력해 주세요"
           />
+          <span class="error-message"></span>
+          </div>
+          <div class="form-group">
           <label>비밀번호</label>
           <input
-            id=""password"
+            id="password"
             name="password"
             type="password"
             placeholder="비밀번호를 입력해 주세요"
           />
-          <button type="submit" onclick="location.href='/sign'" class="loginclick">
+          <span class="error-message"></span>
+          </div>
+          <button type="submit" onclick="location.href='items.html'" id="loginclick">
```

Comment:
P2:
HTML 내부에서 아이디와 클래스 이름의 형식을 통일하는 것이 좋습니다.
지금은 카멜케이스와 케밥케이스가 혼용되어 사용되는 것 같습니다.
클래스명을 케밥케이스로 작성하시니 아이디는 카멜케이스로 작성하시는 것이 더 좋을 것 같네요.

```suggestion
          <button type="submit" onclick="location.href='items.html'" id="loginClick">
```